src/main/resources/db/changelog/0.1.0/test_data/product/products.csv

```diff
@@ -0,0 +1,83 @@
+id,description,price
+d5977f31-5c2e-4f3b-9c7c-5dad32772fbc,AMD Ryzen 5 3600 3.6 - 4.2 GHz - 16 GB RAM - AMD Radeon RX 6600 - 1 TB SSD - Windows,999.99
```

Comment:
Tak wygląda description. Zrobiłem to na wzór opisów jednolinijkowych dla pc, smartfonów oraz tv w sklepie Euro.
Wygenerowałem je na podstawie danych z tabeli (prócz smartfonów i tvs).
Co ciekawe producent iphone nie podaje np. ilości ramu, więc przy generowaniu opisów smartfona, trzeba będzie obsłużyć brak separatora `-`, gdy nie ma jakiejś wartości